swap/peer.go

```diff
@@ -85,7 +85,7 @@ func (p *Peer) setLastSentCheque(cheque *Cheque) error {
 }
 
 func (p *Peer) getLastCumulativePayout() uint64 {
-	lastCheque := p.getLastReceivedCheque()
+	lastCheque := p.getLastSentCheque()
```

Comment:
This function is ambiguous and I would argue that both the `getLastReceivedCheque` and the `getLastSentCheque` variations are correct. Proposing to make two functions out of it:
- `getLastCumulativePayoutTo` and 
- `getLastCumulativePayoutFrom`.

Reply:
I guess I used the wrong function by mistake here while refactoring. `getLastSentCheque` is of course the correct one here as this is used for creating cheques. I agree renaming this would be good to make that clearer.

Reply:
Yes indeed I wanted to ask you @ralph-pichler if I made the correct change.
Without it the tests were failing.

So here a good example why the simulations are worth it!

Reply:
We can create an issue for this and rename in a separate PR (things like this help a PR keep small)